### PR TITLE
Solved Issues #3963 and #3960

### PR DESCRIPTION
**Pull request by joshuazeltser**
Fixed issues #3963 and #3960. For static html add aria-label with ordering and for dynamic elements used javascript to add aria-labels dynamically.

### CODE REVIEW

**Comment by takluyver** [comment]
Is it really necessary to explicitly number things which already have an order in the DOM? That seems like it's bound to get broken as people modify the menus.

File: notebook/static/tree/js/newnotebook.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ define([
                 .attr("id", "kernel-" +ks.name)
                 .data('kernelspec', ks).append(
                     $('<a>')
+                        .attr("aria-label","list item 1,"+ks.name)
```
Comment (takluyver) [inline notebook/static/tree/js/newnotebook.js]: Will this not result in multiple `list item 1`?

Comment (joshuazeltser) [reply notebook/static/tree/js/newnotebook.js]: This has been made dynamic

**Comment by joshuazeltser** [comment]
The specified menus should all now be numbered dynamically by the screenreader

**Comment by takluyver** [comment]
Thanks!

**Comment by takluyver** [comment]
BTW, if you write your commit messages slightly differently, like "Fixes #3963, fixes #3960", the issues get automatically closed when we merge the PR. There are a few synonyms it recognises, but "Solved issues" is not among them:

https://github.blog/2013-01-22-closing-issues-via-commit-messages/

**Pull request merged**